hyperdome_client/hyperdome_client.py

```diff
@@ -58,9 +59,10 @@ def __init__(
         self.app = app
         self.local_only: bool = local_only
         self.common.log("OnionShareGui", "__init__")
+        self.counter = 0
```

Comment:
Maybe give this a more informative name

Reply:
better yet I can remove it because I only had it for debugging a race condition :D